account_analytic_distribution_manual/models/base.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2024 Tecnativa - Carlos Lopez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import json
+
+from lxml import etree
+
+from odoo import api, models
+from odoo.tools.misc import frozendict
+
+
+class AnalyticMixin(models.AbstractModel):
+    _inherit = "base"
+
+    @api.model
+    def get_view(self, view_id=None, view_type="form", **options):
+        def add_field(node, view_type, res_model):
+            attribute = "column_invisible" if view_type == "tree" else "invisible"
+            modifiers = json.dumps({attribute: True})
```

Comment:
Are you sure you need to add manually the modifiers? I haven't needed that on the past.

Reply:
Yes, it's necessary because the modifier for the tree view is different from other views. Otherwise, the field would be shown in the views.